features/definitions_override.feature

```diff
@@ -1,4 +1,5 @@
 Feature: Step Definitions Override
+  Using step definitions as annotations
```

Comment:
I would add this to the `I need to be able to override definition methods using annotations` to keep the `In order to .. as a ... I need to...` three-line structure